dbms/src/Functions/FunctionRegexpQuoteMeta.h

```diff
@@ -0,0 +1,112 @@
+#include <Common/config.h>
```

Comment:
You can move all of this to .cpp, because it's not make sense to include this header somewhere else.